src/lib/components/wizard-navigation-bar.component.html

```diff
@@ -14,7 +14,7 @@
         <ng-container *ngIf="!step.stepTitleTemplate">{{step.stepTitle}}</ng-container>
       </div>
       <div class="step-indicator"
-        [ngStyle]="{ 'font-family': step.stepSymbolTemplate ? '' : step.navigationSymbol.fontFamily }">
+        [style]="{ 'font-family': step.stepSymbolTemplate ? '' : step.navigationSymbol.fontFamily }">
```

Comment:
Angular 9 allows the use of the `style` and `class` bindings in addition to the "legacy" `ngStyle` and `ngClass` directives. See also section "**Improved CSS class and style binding**" in https://blog.angular.io/version-9-of-angular-now-available-project-ivy-has-arrived-23c97b63cfa3